10/calculator.go

```diff
@@ -0,0 +1,22 @@
+package calculator
+
+import "errors"
+
+func Subtract(x int, y int) int {
+	return x - y
+}
+
+func Multiply(x int, y int) int {
+	return x * y
+}
+
+func Divide(x int, y int) (int, error) {
+	if y == 0 {
+		return 0, errors.New("division by zero")
```

Comment:
Even better would be something like:

```suggestion
		return 0, fmt.Errorf("%d / %d: division by zero", x, y)
``